study.md

```diff
@@ -24,7 +24,8 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, how Ember "stops breaking the web".
 
 ```md
-<!-- your answer here -->
+Ember's router uses specific URL's to decide when to send and recieve data from
+the backend. Ember uses consistent URL structure throughout any given Ember page.
```

Comment:
Not exactly. You're right that Ember uses multiple different URLs. In that sense it is different from a traditional SPA, which has one page and _only one URL_. By having multiple URLs, each pointing to separate view states, an Ember app will act like an old-school static web page, something the browser is very comfortable dealing with.